pyproject.toml

```diff
@@ -24,8 +24,7 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.7"
 click = ">=7.1, <9.0"
-# Skipping 1.9.x versions due issue https://github.com/samuelcolvin/pydantic/issues/4114
-pydantic = {version = "~1.8.2", extras = ["dotenv"]}
+pydantic = {version = ">= 1.8.0, != 1.9.*, >= 1.10.4", extras = ["dotenv"]}
```

Comment:
Remind me, does this actually allow both 1.8.x and 1.10.4+? Or would it be equivalent and simpler to just have `">= 1.10.4"` alone?

Reply:
the idea is to keep the parser working for environments using Pydantic pinned in 1.8.X versions, and start using the newer versions

Reply:
Right, just wanted to make sure that's what it's actually doing :)